rosidl_generator_py/cmake/rosidl_generator_py_generate_interfaces.cmake

```diff
@@ -26,24 +26,12 @@ find_package(PythonExtra MODULE)
 # TODO(esteve): force opensplice and connext C type supports only, uncomment
 # the following line when all typesupport implementations are ported to C
 #get_rmw_typesupport(_typesupport_impls ${rmw_implementation})
```

Comment:
I actually think this comment is ok. lines 29-46 here, which were moved into the macro, are a substitute for line 28, `get_rmw_typesupport`, which should be reenabled when all typesupport impls are ported to C. The comment above refers to line 28.